.circleci/config.yml

```diff
@@ -163,4 +163,4 @@ workflows:
   version: 2
   build_and_test:
     jobs:
-      - build_and_tests
\ No newline at end of file
+      - build_and_tests
```

Comment:
Whats the diff in this line? Is the new line missing?

Reply:
That wasn't intentional, removed.  Was trying to see what was off with the tests failing.